packages/node_modules/@node-red/nodes/locales/en-US/storage/10-file.html

```diff
@@ -19,8 +19,6 @@
        Alternatively, it can delete the file.</p>
     <h3>Inputs</h3>
     <dl class="message-properties">
-        <dt class="optional">filename <span class="property-type">string</span></dt>
-        <dd>If not configured in the node, this optional property sets the name of the file to be updated.</dd>
```

Comment:
We should not remove all mention that the filename can be provided as a message property. I cannot propose changes for deleted lines in the normal github way, but I'd suggest changing the text to something like:

```
        <dt class="optional">filename <span class="property-type">string</span></dt>
        <dd>The name of the file to be updated can be provided in the node configuration, or as a message property. By default it will use `msg.filename` but this can be customised in the node.</dd>
```

Reply:
fixed in 4fffa2d